content/installation/node/NodeSupportedTechnologies.md

```diff
@@ -4,30 +4,44 @@ description: "List of supported technologies"
 tags: "installation Node agent frameworks support troubleshooting nodejs javascript modules"
 -->
 
-Contrast supports Node versions 6 and 8 Long-Term Support (LTS). Contrast shifts support for Node versions as the Node project shifts its LTS windows. For a schedule, see the [Node.js Long-Term Support Release Schedule](https://github.com/nodejs/LTS).
+## Node.js Version Support
 
-Third-Party Module Support                                     | Web Framework Support<sup>[1](#footnote1)</sup>           | Coming Soon
-:-----                                                         | :----                                                     | :-------
-[express-session](https://github.com/expressjs/session)        | [express](http://expressjs.com)                           | [koa](https://www.npmjs.com/package/koa)
-[mongodb](https://docs.mongodb.org/ecosystem/drivers/node-js/) | [sails](http://sailsjs.org/)                              |
-[mysql](https://www.npmjs.com/package/mysql)                   | [hapi](https://www.npmjs.com/package/hapi) (version 16.x) |
-[postgres](https://www.npmjs.com/package/pg)                   |                                                           |
+Contrast supports Node.js Long-Term Support (LTS) versions 6, 8 and 10. Contrast shifts its support for Node.js versions as the working group shifts its LTS windows. For the schedule, see the [Node.js Long-Term Support Release Schedule](https://github.com/nodejs/LTS).
+
+> **Note:** The Node.js agent currently doesn't support HTTP2, and generally doesn't support Node.js features classified as Experimental (Stability: 1).
+
+## Web Framework Support
+
+* [Express](http://expressjs.com) version 4
+* [hapi](https://www.npmjs.com/package/hapi) versions 16 and 17
+* [Sails](http://sailsjs.org/) version 0.12
+
+While the agent can still run on web frameworks that aren't officially supported, Contrast may produce less-specific findings than it would for supported frameworks. Instead of reporting that a vulnerability occurs in your application code, Contrast may need to report it within the framework code where it interfaces directly with Node.js's built-in [http](https://nodejs.org/api/http.html) module.
+
+
+## Third-Party Module Support
+
+* [express-session](https://github.com/expressjs/session)
+* [MongoDB](https://docs.mongodb.org/ecosystem/drivers/node-js/)
+* [MySQL](https://www.npmjs.com/package/mysql)
+* [Mongoose](http://docs.sequelizejs.com/)
+* [postgres](https://www.npmjs.com/package/pg)
+* [Sequelize](http://docs.sequelizejs.com/)
```

Comment:
Sequelize and Mongoose have the same hyperlinks. Is that correct?

Reply:
Good catch. Very wrong.

Reply:
FIXED!